src/frontend/src/reducers/neighborhoods/selectors/drawNeighborhoodRoutes.ts

```diff
@@ -0,0 +1,187 @@
+// Refactored from old codebase at latest commit before introducing listings: 8ca3b94
+// New code copied from latest Taui v3 to support routes with MapLibre
+// ATTR: Taui by Conveyal, included under the MIT license (https://github.com/conveyal/taui/blob/dev/LICENSE)
+
```

Comment:
This file is essentially the same as old drawNeighborhoodRoute except now it's wrapped in a function to allow passing in dynamic destination and neighborhood identifiers. Previously this selector pulled the active destination and active neighborhood from state, but we need 1) routes for each destination for an active neighborhood in the details view 2)  routes for each destination for every favorited neighborhood in the compare view. This allows creating a destination-->neighborhood route for any given combo